conftest.py

```diff
@@ -1,18 +1,6 @@
-from django import http, test
-from django.core.cache import caches
-from django.utils import translation
-
 import responses
 import pytest
 
-from multidb import pinning
-
-from olympia import amo, core
-from olympia.access.models import Group, GroupUser
-from olympia.amo.tests import start_es_mocks, stop_es_mocks
-from olympia.translations.hold import clean_translations
-from olympia.users.models import UserProfile
-
```

Comment:
Include a comment at the top explaining the strategy regarding the django-related imports, so that we don't try to add some by accident later